### PR TITLE
Docstring lint change

### LINKED ISSUE #199: Enable D202 and D204 for flake8-docstrings
In a discussion on #dev-contrib, it was decided to no longer ignore these across all repos. See https://github.com/python-discord/organisation/issues/53#issuecomment-486371508 for more information.

Remove these from the ignore list, lint, and open a pull request.

@sco1 said he'd do this when the open PRs are merged, so I'm assigning it to him.

### PR DESCRIPTION
**Pull request by sco1**
Remove D202, D204 from ignored linting codes & relint the repository

Closes #199

### CODE REVIEW

**Comment by sco1** [comment]
This has been synced to `master` and opened for review in advance of the upcoming Pride season.

Existing PRs will need to be relinted when updated for current review comments.

File: bot/seasons/easter/egg_hunt/cog.py [hunk 1]
```diff
@@ -156,23 +150,21 @@ def finalise_score(self):
...
-
         if user.bot:
             return False
         if reaction.message.id != self.message.id:
             return False
         if reaction.emoji != self.egg:
             return False
 
-        # ignore the pushished
+        # Ignore the pushished
```
Comment (lemonsaurus) [inline bot/seasons/easter/egg_hunt/cog.py]: what's `pushished`?

Comment (sco1) [reply bot/seasons/easter/egg_hunt/cog.py]: I figured it was some Australian thing

Comment (Suhail6inkling) [reply bot/seasons/easter/egg_hunt/cog.py]: Is it not supposed to be `punished`?

**Comment by sco1** [comment]
- Updated to address pre-commit's misconfiguration resulting in it not actually using any of our flake8 plugins.
- Fix null bytes error with the `.snakes draw` command. Closes #218

**Pull request merged**